README.md

```diff
@@ -49,3 +49,25 @@ You can also install it as a
 cd scripts/
 make install
 ```
+
+### Testing the Database Locally
+
+Some tests must also be run on the DynamoDB database. We recommend that you
+download it and keep it running in the background while executing tests.
+
+```bash
+wget https://s3-us-west-2.amazonaws.com/dynamodb-local/dynamodb_local_latest.tar.gz
+mkdir DynamoDB
+tar -xvf dynamodb_local_latest.tar.gz --directory DynamoDB
+
+# Configure AWS
+scripts/setup_localaws.sh
+
+# Run DynamoDB through Java
+cd DynamoDB/
+java -Djava.library.path=./DynamoDBLocal_lib -jar DynamoDBLocal.jar -sharedDb
+```
+
+To have it executing in the background, simply add an ampersand (&) at the end
```

Comment:
It might be useful to add slightly more detail here for users that don't know much about running things in the background (i.e. what does it mean to kill a process?). That might be too complex for this though.